src/web-server.js

```diff
@@ -110,6 +113,7 @@ app.patch('/services', function(req, res) {
     if (!error && response.statusCode == 200) {
       modify.modifyService(env, service, url)
       .then((json) => {
+        console.log('sending response json', json)
```

Comment:
I'm guessing this was testing code left in

Reply:
👍 good catch. Updated